### PR TITLE
Add some notes about Docker security

### PR DESCRIPTION
**Pull request by kimmoahokas**
Add really brief notes about Docker security (or lack of it).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -172,6 +173,16 @@ Be aware of possible security threats and problems. You should at least be famil
 
 Good generic security guidelines would be:
 
+## Docker
+
+**Using Docker will not make your service more secure.** Generally, you should consider at least following things if using Docker:
+
+- Don't run any untrusted binaries inside Docker containers
+- Create unprivileged users inside Docker containers and run binaries using unprivileged user instead of root whenever possible
+- Periodically update the dependencies inside Docker containers
```
Comment (ilkka) [inline README.md]: Does this mean we would be advocating doing the equivalent of `apt-get upgrade` within a long-running container? Or does it mean that you should rebuild your images when there are security patches for any libraries or tools they contain?

File: README.md [hunk 2]
```diff
@@ -172,6 +173,16 @@ Be aware of possible security threats and problems. You should at least be famil
...
+- Periodically update your Docker hosts
+- Multiple containers running on same host will have some level of access to other containers and the host itself. Properly secure all containers and hosts
```
Comment (ilkka) [inline README.md]: Could we be a bit more specific about securing? For example, containers that don't need the network can run without any network access, capabilities can be removed etc.

File: README.md (hunk 1 above)
Comment (kimmoahokas) [reply README.md]: I think it's better practice to rebuild the containers, not to upgrade them on-the-fly. I'll clarify that aspect.

File: README.md (hunk 2 above)
Comment (kimmoahokas) [reply README.md]: We definitely could and should be more specific. However, this is the part where my knowledge ends :(

File: README.md (hunk 1 above)
Comment (ilkka) [reply README.md]: Yeah, agree 👍

File: README.md (hunk 2 above)
Comment (ilkka) [reply README.md]: I'll contribute some words to this PR about that :)

**Pull request merged**